nix/outputs.nix

```diff
@@ -0,0 +1,17 @@
+{ repoRoot, inputs, pkgs, lib, system }:
+
+let
+
+  project = repoRoot.nix.project;
+
+  isabelle = import ./isabelle.nix { inherit repoRoot inputs pkgs lib system; };
+
+in
+
+[
+  project.flake
+  {
+    packages.isabelle-test = isabelle.isabelle-test;
+    hydraJobs.packages.isabelle-test = isabelle.isabelle-test;
```

Comment:
@zeme-wana, it looks like the old hydra configuration might have limited to Linux. Is there a way to limit it here, too?